This makes the go agent aware of the added plugins

diff --git a/src/go/plugins/plugins_linux.go b/src/go/plugins/plugins_linux.go
index 714f783..7a57116 100644
--- a/src/go/plugins/plugins_linux.go
+++ b/src/go/plugins/plugins_linux.go
@@ -42,4 +42,5 @@ import (
 	_ "zabbix.com/plugins/zabbix/async"
 	_ "zabbix.com/plugins/zabbix/stats"
 	_ "zabbix.com/plugins/zabbix/sync"
+	_ "zabbix.com/plugins/alpine"
 )
